src/component/feature/FeatureToggleList/FeatureToggleListTable.tsx

```diff
@@ -210,7 +214,11 @@ export const FeatureToggleListTable: VFC = () => {
             isLoading={loading}
             header={
                 <PageHeader
-                    title={`Feature toggles (${data.length})`}
+                    title={`Feature toggles (${
+                        rows.length < data.length
+                            ? `${rows.length} of ${data.length}`
+                            : data.length
+                    })`}
```

Comment:
We should align this with @NicolaeUnleash and eventually apply this to all tables.